tests/deterministic_graph_data.py

```diff
@@ -130,6 +130,8 @@ def create_configuration(
         knn.fit(positions, node_feature)
         node_output_x = torch.Tensor(knn.predict(positions))
 
+    node_output_x = node_output_x + node_feature
+
     node_output_x_square = node_output_x ** 2
```

Comment:
Could you try something like below?
```suggestion
    node_output_x_square = node_output_x ** 2 + node_feature
```

Reply:
I already did. But that one also fails.